stable/firehose/templates/deployment.yaml

```diff
@@ -36,8 +36,8 @@ spec:
           name: workdir
         {{- end }}
         {{- if (gt (len .Values.mountSecrets) 0 ) }}
-        - mountPath: /etc/secret/{{ include "firehose.fullname" . }}-mount-secrets
-          name: {{ include "firehose.fullname" . }}-mount-secrets
+        - mountPath: /etc/secret/{{ include "firehose.fullname" . }}-m-secret
```

Comment:
Let's get rid of `-secret` as well

Reply:
the name would be of 53 chars, so we set the suffix as `-mount`. wdys?